docs/ethpm.rst

```diff
@@ -606,7 +606,10 @@ To inline the source code directly in the manifest, use ``inline_source()`` or `
 
 To include the source as a content-addressed URI, ``Py-EthPM`` can pin your source via the Infura IPFS API. As well as the contract name and compiler output, this function requires that you provide the desired IPFS backend to pin the contract sources.
 
+..
 .. doctest::
+..
```

Comment:
After running `make docs`, this code block isn't showing up. I think we can just remove the doctest for now. 

Here's what it looks like for me: 
<img width="737" alt="image" src="https://user-images.githubusercontent.com/6540608/184172781-76d1b145-948c-4c9a-84b4-400c79a9d901.png">

Reply:
Ah, nice catch. Yeah I'll remove it 👍🏼. The commit should keep track of all that changed.